lib/hz2600/CallflowBuilder/Builder.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace CallflowBuilder;
+
+use CallflowBuilder\Node\AbstractNode; 
+
+class Builder {
```

Comment:
I don't see the point of this class. If we take the $user_node from the docs, why call `Builder::build($user_node)` only to call $user_node->build() within the build/1 function? Why wouldn't I just do `$callflow->flow = $user_node->build()` ?